components/clientComponents/forms/FormattedDate/FormattedDate.cy.tsx

```diff
@@ -177,29 +177,4 @@ describe("<FormattedDate />", () => {
     cy.get("[data-testid=description]").should("exist");
     cy.get("[data-testid=description]").contains("This is a description");
   });
-
```

Comment:
Why was this removed?

Reply:
The test no longer applies because the max/min attributes were removed from the formatted date inputs

Reply:
@thiessenp-cds Dave pushed a commit --- there is error handling in place when you submit.

Please open an issue to create a new Cypress test that will capture the error for a leap year either at the summary or field level.

Reply:
Never mind looks like Dave does have some unit tests for this.